lib/utils/scenario-manager.js

```diff
@@ -47,7 +47,12 @@ module.exports = CoreObject.extend({
     var pkgs = Object.keys(scenario.dependencies);
 
     pkgs.forEach(function(pkg){
-      bowerJSON.dependencies[pkg] = scenario.dependencies[pkg];
+      if (bowerJSON.devDependencies[pkg]) {
+        bowerJSON.devDependencies[pkg] = scenario.dependencies[pkg];
```

Comment:
@truenorth - Agreed.

@martndemus - Would you mind updating for that structure?